menpo/io/input/base.py

```diff
@@ -342,24 +375,38 @@ def import_video(filepath, landmark_resolver=same_name_video, normalize=None,
                    importer_kwargs=kwargs)
 
 
-def import_landmark_file(filepath, asset=None):
+def import_landmark_file(filepath, group=None, asset=None):
     r"""Single landmark file importer.
 
-    If a landmark file is found at ``filepath``, returns a :map:`PointCloud` or
-    :map:`LabelledPointUndirectedGraph` depending on the format of the
-    landmark file.
+    If a landmark file is found at ``filepath``, returns a dictionary
+    of landmarks where keys are the group names and the values are 
+    :map:`PointCloud` or subclasses. If the optional ``group`` argument is
+    supplied then a single group with the given name is returned rather than
+    a dictionary
 
     Parameters
     ----------
     filepath : `pathlib.Path` or `str`
         A relative or absolute filepath to an landmark file.
+    group : `str`, optional
+        The name of the landmark group to return from the landmark dictionary.
+        If None, then a dictionary is returned where keys are the group names
+        and the values are :map:`PointCloud` or subclasses.
+    asset : `object`, optional
+        The object the landmark belongs to (useful for things like rescaling)
 
     Returns
     -------
-    landmarks : :map:`LabelledPointUndirectedGraph` or :map:`PointCloud`
-        The shape that the file format represents.
+    landmarks : `dict` {`str`: :map:`PointCloud`} or :map:`PointCloud`
+        Dictionary mapping landmark groups to :map:`PointCloud` or subclasses
+        OR
+        :map:`PointCloud` or subclass is ``group == None``
```

Comment:
Is this correct? 
If I understand well from the code and my experimentation, you return: 
a) a dictionary in case the group == None, 
b) a PointCloud in case group == [name]. 
Yet i believe the documentation claims the reverse.